src/commands/distribute/release.ts

```diff
@@ -39,50 +45,46 @@ export default class ReleaseBinaryCommand extends AppCommand {
   public releaseNotesFile: string;
 
   public async run(client: AppCenterClient): Promise<CommandResult> {
+
+    const bar = new ProgressBar(' Uploading a new release [:bar] :percent', {
```

Comment:
How does this progress bar integrate with the `--quiet` switch? Looks like it doesn't. You shouldn't display the progress bar if `--quiet` is given. Look at the stuff in the interaction directory (like out.text, out.progress, etc.) to see the patterns used to handle this.

Reply:
that is a really good point, have abstracted the method.